packages/tradershub/src/components/ModalStepWrapper/ModalStepWrapper.tsx

```diff
@@ -42,28 +42,26 @@ const ModalStepWrapper: FC<PropsWithChildren<TModalStepWrapperProps>> = ({
 
     return (
         <div
-            className={classNames('wallets-modal-step-wrapper', {
-                'wallets-modal-step-wrapper--fixed-footer': fixedFooter && !shouldHideHeader,
-                'wallets-modal-step-wrapper--hide-deriv-app-header': shouldHideDerivAppHeader,
-                'wallets-modal-step-wrapper--no-header': shouldHideHeader && !fixedFooter,
-                'wallets-modal-step-wrapper--no-header--fixed-footer': shouldHideHeader && fixedFooter,
+            className={classNames('modal-step-wrapper', {
+                'modal-step-wrapper--fixed-footer': fixedFooter && !shouldHideHeader,
+                'modal-step-wrapper--hide-deriv-app-header': shouldHideDerivAppHeader,
+                'modal-step-wrapper--no-header': shouldHideHeader && !fixedFooter,
+                'modal-step-wrapper--no-header--fixed-footer': shouldHideHeader && fixedFooter,
```

Comment:
I feel this can be improved. The combination we have whether it should have Header or should have Footer is already very confusing. If we choose the compound components approach it will be better. Take a look at the `<Dialog />` component that we have in the `tradershub` package as reference.